conf/app.ini

```diff
@@ -439,6 +439,11 @@ SCHEDULE = @every 24h
 ; Archives created more than OLDER_THAN ago are subject to deletion
 OLDER_THAN = 24h
 
+; Synchronize external user data
+[cron.sync_external_users]
+SCHEDULE = @every 24h
+UPDATE_EXISTING = true
```

Comment:
I actually recently found out that's not the case, there's an internal default in `setting.go` for all the settings, so that `conf/app.ini` really only has the role of a template to be copied to the `custom/conf/` directory, for customizing it. When customizing values (changing them) it's still useful to have comments telling what the default is.